gatling-recorder/src/main/scala/io/gatling/recorder/har/HarReader.scala

```diff
@@ -58,17 +60,26 @@ private[recorder] object HarReader {
   private def createRequestWithArrivalTime(entry: Entry): TimedScenarioElement[RequestElement] = {
       def buildContent(postParams: Seq[PostParam]): RequestBody =
         RequestBodyParams(postParams.map(postParam => (postParam.name, postParam.value)).toList)
+      def decode(s: String): String = URLDecoder.decode(s, UTF8.name)
 
     val uri = entry.request.url
     val method = entry.request.method
     val requestHeaders = buildRequestHeaders(entry)
 
     // NetExport doesn't copy post params to text field
     val requestBody = entry.request.postData.flatMap { postData =>
-      if (postData.params.nonEmpty)
-        Some(buildContent(postData.params))
-      else
+      if (postData.params.nonEmpty) {
+        val requestContentType = requestHeaders.get(ContentType)
+        val isUrlEncoded = requestContentType.exists(_.contains(ApplicationFormUrlEncoded))
```

Comment:
copy-paste from `ScenarioElement`...
I think `equalsIgnoreCase` is not the right method to use here because the HTTP Content-Type header contains a media-type value (https://tools.ietf.org/html/rfc7231#section-3.1.1.5), which can contain parameters (https://tools.ietf.org/html/rfc7231#section-3.1.1.1)